env/prod.env

```diff
@@ -1 +1 @@
-IM_API=https://api.im.prod.test.c.dhis2.org
+IM_API=https://api.im.dev.test.c.dhis2.org
```

Comment:
This isn't right :-) The file is called prod but now contains a url to dev. I think we should target dev by default.